point_cloud_transport/include/point_cloud_transport/publisher_plugin.hpp

```diff
@@ -67,7 +68,7 @@ class PublisherPlugin
   //! \brief Advertise a topic, simple version.
   POINT_CLOUD_TRANSPORT_PUBLIC
   void advertise(
-    std::shared_ptr<rclcpp::Node> node,
+    NodeInterfaces::SharedPtr node_interfaces,
```

Comment:
we should deprecate the old methods, we apply tic-toc, we deprecate the method in the release and we will remove them in the next one. 

Same below